samples/matter/common/src/dfu/smp/dfu_over_smp.cpp

```diff
@@ -30,29 +32,36 @@ constexpr uint16_t kAdvertisingIntervalMin = 400;
 constexpr uint16_t kAdvertisingIntervalMax = 500;
 constexpr uint8_t kAdvertisingFlags = BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR;
 
+static bool sDfuInProgress = false;
+static uint32_t sDfuSyncMutexId;
+
 namespace
 {
-enum mgmt_cb_return UploadConfirmHandler(uint32_t event,
-					 enum mgmt_cb_return prev_status,
-					 int32_t *rc, uint16_t *group,
-					 bool *abort_more, void *data,
-					 size_t data_size)
+enum mgmt_cb_return UploadConfirmHandler(uint32_t event, enum mgmt_cb_return prev_status, int32_t *rc, uint16_t *group,
```

Comment:
@markaj-nordic  it's yet another place where we can have different clang-format configs.

Reply:
So is it correct now, or my configuration is wrong?

Reply:
Not sure, maybe the formatting here was invalid in the first place.